sml_builder/utils.py

```diff
@@ -3,6 +3,14 @@
 from flask import abort, url_for
 
 
+def checkTypeList(data):
```

Comment:
Not sure why this is showing as new, currently this is same as in main

Reply:
checkTypeList and checkEmptyList look like they've been defined twice when I pull your branch

Reply:
good catch not sure why my merge has decided thats the right way to do it, amended